easybuild/easyconfigs/n/numba/llvmlite-0.22.0_fix-ffi-Makefile.patch

```diff
@@ -0,0 +1,15 @@
+--- llvmlite-0.22.0/ffi/Makefile.linux.orig	2018-02-15 23:41:57.000000000 +0100
```

Comment:
@smoors Please include short description + reference to upstream source or author at the top of the patch